sources/models/src/lib.rs

```diff
@@ -216,6 +216,7 @@ struct KubernetesSettings {
     image_gc_high_threshold_percent: ImageGCHighThresholdPercent,
     image_gc_low_threshold_percent: ImageGCLowThresholdPercent,
     provider_id: Url,
+    kubelet_log_level: i32,
```

Comment:
Is `i32` the right type? Does `kubelet` handle negative or very large values, or does it expect it to be capped in some way?

Reply:
Perhaps this was laziness, but I used `i32` just based on other settings we had. In reality, the number should only be between 0 and 9. I wonder if this would be worth adding a custom type to help enforce? I don't believe it would cause any problems if a user set it to something like -12, but it would help make it clear what the actual valid values should be.